lib/Client.js

```diff
@@ -179,10 +178,20 @@ class Client extends EventEmitter {
         this.reconnectAttempts = 0;
     }
 
+    get token() {
```

Comment:
This getter seems entirely useless outside of backwards compatibility, but I see no real use for that here.

Reply:
The conversation we had about that: <https://discordapp.com/channels/81384788765712384/381890266052689920/687247603381043210>

I have personally no issue with using `client._token` in Shard and RequestHandler instead of the getter, but maybe other point of view would be nice before changing the implementation?
I'll end up doing what Eljay suggested to simplify the code unless other poeple vote against that.

Reply:
Seeing as `Client#token` is documented, it should remain in some form, regardless of your beliefs on how strictly we should adhere to its private nature.